framework/requirements/requirements.php

```diff
@@ -73,6 +73,17 @@
         INTL_ICU_VERSION . '.'
     ),
     array(
+        'name' => 'ICU Data version',
+        'mandatory' => false,
```

Comment:
I think we can switch `mandatory` to `true` in both ICU and ICU Data requirements